### DISCUSSION TITLE
Problem on Modal Tag

### DISCUSSION THREAD

**Discussion by ryanadiyantara** [chakra-ui/chakra-ui#9316] [Q&A] [answered]
I have a problem in the ModalOverlay tag where on each page the bg color always has a different opacity. I've searched and tried deleting other components, but still nothing affects it. Likewise with other settings such as boxShadow and backdropFilter which will set themselves on different pages, even though I created the modal in the component.

![1](https://github.com/user-attachments/assets/1f1ac4cf-5a55-4538-bc6f-695eaeaf43be)
![2](https://github.com/user-attachments/assets/478e3225-b9dd-43a9-8696-a25dfa928605)
![3](https://github.com/user-attachments/assets/5f2df0e5-a3bf-48e1-bcd5-25965e8a041f)

Here is the code of my modal component:
[
![4](https://github.com/user-attachments/assets/5ba2795a-48f5-487d-ab27-93eb68bc5139)
](url)

**Comment by segunadebayo** [comment, marked as answer]
Modal is `Dialog`. Style `Dialog.Backdrop`.
